client/state/current-user/email-verification/reducer.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import { createReducer } from 'state/utils';
+import {
+	EMAIL_VERIFY_REQUEST,
+	EMAIL_VERIFY_REQUEST_SUCCESS,
+	EMAIL_VERIFY_REQUEST_FAILURE,
+} from 'state/action-types';
+
+export const status = createReducer( null, {
+	[ EMAIL_VERIFY_REQUEST ]: () => 'requesting',
+	[ EMAIL_VERIFY_REQUEST_SUCCESS ]: () => 'sent',
+	[ EMAIL_VERIFY_REQUEST_FAILURE ]: () => 'error',
```

Comment:
@v18 The reducers don't appear to be listening to `EMAIL_VERIFY_STATE_RESET`. If there's an unknown type being dispatched each reducer will return initial state, but only if `state` is undefined, otherwise the default is to return current state.